src/backend/wine/manager/utils.ts

```diff
@@ -88,6 +89,7 @@ async function updateWineVersionInfos(
   }
 
   logInfo('wine versions updated', { prefix: LogPrefix.WineDownloader })
+  BrowserWindow.getAllWindows()[0].webContents.send('wineVersionsUpdated')
```

Comment:
Shame we can't use `getMainWindow()` here (I assume Jest doesn't like all of our aliased imports in `utils`)

Reply:
I'm planning to refactor all these calls creating something like a `frontendEvents` module with `send` function to hide this implementation detail so we can use it always instead, it will also help with testing (to test something like this we have to mock 3 different modules for example)